src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -235,6 +235,33 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 		setHideLowBalance(!hideLowBalance);
 	};
 
+	const addToMetamask = async (address: string, symbol: string, token: any) => {
+		if (symbol.length >= 11) {
+			symbol = symbol.slice(4, symbol.length);
+		}
+		//@ts-ignore
+		const url = iconsCoin[token.coin];
+		const decimals = 18;
```

Comment:
This can be retrieved from token object otherwise just set it in the options object below, since it's a constant and that would decrease the lines of code we need to write

Reply:
done